### PR TITLE
Translations for Trial Org Metadata

### PR DESCRIPTION
**Pull request by force2b**
- Add translations for the following metadata that is included in the Trial Org Configuration:
   - Opportunity.RecordType Labels and Description
   - Opportunity.Type Pick List Values
   - Opportunity.StageName Pick List Value
   - Opportunity Contact Role Pick List Values
- Add a new CCI task for `deploy_trial_translations` that will deploy translations for the above metadata into a target org.

# Critical Changes

# Changes

# Issues Closed

# New Metadata

# Deleted Metadata

# Unaggregated Pull Requests
* Loc Delivery Unpackaged Trial Org 2020/01/10 [[PR4946](https://github.com/SalesforceFoundation/NPSP/pull/4946)]

### CODE REVIEW

**Comment by force2b** [comment]
**lurch: attach W-038417

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x1E000001HqIEQA0" target="_blank">W-038417</a>

**Comment by force2b** [comment]
**lurch: attach W-038417

File: unpackaged/config/trial_translations/objectTranslations/Opportunity-de.objectTranslation [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    <label>Zugewiesene Schenkung</label>
+    <name>MatchingGift</name>
+  </recordTypes>
+  <recordTypes>
+    <description>Ein Mitgliedschaft-Datensatztyp</description>
+    <label>Mitgliedschaft</label>
+    <name>Membership</name>
+  </recordTypes>
+  <layouts>
+    <layout>___NAMESPACE___Major Gift Layout</layout>
```
Comment (davidmreed) [inline unpackaged/config/trial_translations/objectTranslations/Opportunity-de.objectTranslation]: The token `___NAMESPACE___` is replaced only in filenames. In the file body, it needs to be `%%%NAMESPACE%%%`.

Comment (force2b) [reply unpackaged/config/trial_translations/objectTranslations/Opportunity-de.objectTranslation]: Thanks. Changes made.

**Comment by rponti-sforg** [comment]
@force2b I have done a rebuild (twice) on the Feature Test - Robot tests since ten of the tests were failing.  Looking at the test failures, they are all similar.  The errors involve an 'element' not being found within a specific amount of time.  These are consistent to what QE was observing during the last release automations and likely due to changes made to locators.  Just let me know if you want me to go ahead with merge or if further investigation is needed.

**Comment by force2b** [comment]
@rponti-sforg Thanks. You can ignore the robot errors.

**Comment by rponti-sforg** [comment]
QE testing is complete for this PR and corresponding WI.  Tests have passed.

For further testing details, see:
https://foundation.lightning.force.com/lightning/r/agf__ADM_Work__c/a2x1E000001HqIEQA0/view
https://salesforce.quip.com/uAFGAMmdPbOr

Merging PR.  WI status set to 'Closed' once post PR merge builds are complete (successful).

**Pull request merged**